### ISSUE TITLE
Missing min six version dependency

### ISSUE THREAD

**Issue by dustinfarris** [pytest-dev/pytest-bdd#128]
On Python 2.7, we have six pegged to a version that apparently does not have tools needed by the latest pytest-bdd.

We will probably upgrade our six install; but nonetheless, pytest-bdd should probably declare a minimum requirement.

```console
$ pip install --upgrade pytest-bdd

Downloading/unpacking pytest-bdd
  Downloading pytest-bdd-2.11.1.tar.gz (55kB): 55kB downloaded
  Running setup.py (path:/Users/dustin/.virtualenvs/napa/build/pytest-bdd/setup.py) egg_info for package pytest-bdd
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/Users/dustin/.virtualenvs/napa/build/pytest-bdd/setup.py", line 11, in <module>
        import pytest_bdd
      File "pytest_bdd/__init__.py", line 6, in <module>
        from pytest_bdd.steps import given, when, then
      File "pytest_bdd/steps.py", line 43, in <module>
        from .feature import parse_line, force_encode
      File "pytest_bdd/feature.py", line 39, in <module>
        @six.python_2_unicode_compatible
    AttributeError: 'module' object has no attribute 'python_2_unicode_compatible'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/Users/dustin/.virtualenvs/napa/build/pytest-bdd/setup.py", line 11, in <module>

    import pytest_bdd

  File "pytest_bdd/__init__.py", line 6, in <module>

    from pytest_bdd.steps import given, when, then

  File "pytest_bdd/steps.py", line 43, in <module>

    from .feature import parse_line, force_encode

  File "pytest_bdd/feature.py", line 39, in <module>

    @six.python_2_unicode_compatible

AttributeError: 'module' object has no attribute 'python_2_unicode_compatible'
```

**Comment by bubenkoff** [comment]
Pr is welcome

**Comment by bubenkoff** [comment]
that's fixed in 2.11.3

**Comment by dustinfarris** [comment]
:+1:
